src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
```

Comment:
Consider using try-with-resources for managing the `Session` and `Transaction` to ensure they are closed properly, even if exceptions occur. This will help maintain resource integrity and reduce boilerplate code.